### ISSUE TITLE
Fail clean-resources jenkins job if there is an error

### ISSUE THREAD

**Issue by pehala** [scylladb/scylla-cluster-tests#9544]
Based on the same symptom as #9543, clean-resources jenkins jobs is green https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-clean-test-resources/76/) . We should fail the job as well, so we know something is wrong

**Comment by fruch** [comment]
from day one of Argus, all reports to Argus were defined non critical, and shouldn't be stop any test flow.
so no bug/issue of argus would interfere with the run of the test (or any other parts of the pipeline)

even if we will want to change some place to be reported, we should be that very very carefully, we should want to stop the cleanup in the middle, what ever happening.

**Comment by pehala** [comment]
Agreed, so currently, if the clean-up itself fails, the job should as well?

**Comment by fruch** [comment]
@pehala anything else with this one ?
